docs/extensions/guides/protocol-handlers.md

```diff
@@ -0,0 +1,57 @@
+# Lens Protocol Handlers
+
+Lens has a file association with the `lens://` protocol.
+This means that Lens can be opened by external programs by providing a link that has `lens` as its protocol.
+Lens provides a routing mechanism that extensions can use to register custom handlers.
+
+## Registering A Protocol Handler
+
+The field `protocolhandlers` exists both on [`LensMainExtension`](extensions/api/classes/lensmainextension/#protocolhandlers) and on [`LensRendererExtension`](extensions/api/classes/lensrendererextension/#protocolhandlers).
```

Comment:
It is, but the api docs generator makes everything lowercase because URLs are case-insensitive.

Reply:
But this isn't in a URL. See "The field -> protocolhandlers <- exists both on..."

Reply:
True, however it is an "inter-doc" link pointing to a generated markdown file. I figured out what it would be by looking at https://docs.k8slens.dev/latest/extensions/api/classes/lensmainextension/#appmenus which is also all lower case even though if you go to it you will see that the field itself is `appMenus`

Reply:
🤔 I'm not sure we're looking at the same piece of text. I acknowledge that the _anchor_ or _fragment_ in `lensmainextension/#protocolhandlers` and `lensrendererextension/#protocolhandlers` will be all lowercase, but you're telling me that the property name in plain text that precedes it in `` "The field `protocolhandlers` exists both on" `` also has to be lowercase because that somehow becomes a link? I don't mean to press the issue; just seems strange, and looks like a typo, but if that really is the case, then okidoki. 🤷‍♂️

Reply:
oh, sorry my bad.